source/Calamari.Tests/Fixtures/Deployment/Azure/OctopusTestAzureSubscription.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+using Calamari.Deployment;
+using NUnit.Framework;
+using Octostache;
+
+namespace Calamari.Tests.Fixtures.Deployment.Azure
+{
+    public class OctopusTestAzureSubscription
+    {
+        public const string AzureSubscriptionId = "8affaa7d-3d74-427c-93c5-2d7f6a16e754";
```

Comment:
Yup.  And if the certificate is not installed on the machine, the test will be ignored.